docs/platforms/python/integrations/celery/crons.mdx

```diff
@@ -62,7 +62,7 @@ Once Sentry Crons is set up, tasks in your Celery beat schedule will be auto-dis
 
 Start your Celery beat and worker services and see your tasks being monitored at [https://sentry.io/crons/](https://sentry.io/crons/).
 
-Currently, only the default scheduler of Celery Beat is officially supported. Other schedulers (like `RedBeat`) are not supported.
+In addition to the default Celery Beat scheduler we also support Redbeat.
```

Comment:
```suggestion
In addition to the default Celery Beat scheduler, we also support Redbeat.
```
Is there any link we can add for Redbeat to give users more info?

Reply:
I'll add one.